src/Caching/Test/ClassesTests.cs

```diff
@@ -150,5 +150,21 @@ class B(A):
             // In JSON, class A should have 'class A doc' documentation while B should have none.
             Baseline.CompareToFile(BaselineFileName, json);
         }
+
+         [TestMethod, Priority(0)]
+        public void ClassesWithSuper() {
+//            const string code = @"
+//class A:
+//    def methodA(self):
+//        return True
+
+//class B(A):
+//    def methodB(self);
+//        x = super()
```

Comment:
The persistence I would be testing here is `return super()`, since `x` is a local variable that will not be persisted.

Reply:
thanks updated